modules/renter.go

```diff
@@ -92,10 +92,35 @@ type RenterFinancialMetrics struct {
 	DownloadSpending types.Currency `json:"downloadspending"`
 	StorageSpending  types.Currency `json:"storagespending"`
 	UploadSpending   types.Currency `json:"uploadspending"`
+}
+
+// RenterPeriodMetrics contains metrics relevant to an allowance period.
+type RenterPeriodMetrics struct {
+	// The specified Allowance for the period, and the height at which it went
+	// into effect. If the allowance was changed mid-period, this field
+	// reflects the most recent version. The PeriodStart, however, is
+	// unchanged.
+	Allowance   Allowance         `json:"allowance"`
+	PeriodStart types.BlockHeight `json:"periodstart"`
+
+	// The amount of money spent on fees during the period.
+	TxnFeeSpending      types.Currency `json:"txnfeespending"`
+	ContractFeeSpending types.Currency `json:"contractfeespending"`
+	SiafundFeeSpending  types.Currency `json:"siafundfeespending"`
+
+	// The amount of money allocated for uploads, downloads, and storage
+	// during the period.
+	ContractAllocations types.Currency `json:"contractallocations"`
```

Comment:
Meaning, money that was locked into transactions? Comment string can probably be made a little more clear.